runconfig/parse_windows.go

```diff
@@ -0,0 +1,20 @@
+package runconfig
+
+import (
+	"fmt"
+	"strings"
+)
+
+func parseNetMode(netMode string) (NetworkMode, error) {
+	parts := strings.Split(netMode, ":")
```

Comment:
What is the purpose of `default` mode to have multiple parts ?
In the case of Linux, multiple parts separated by ":" is used for container namespace. which is not applicable for windows.